pkg/api/testing/fuzzer.go

```diff
@@ -111,4 +112,5 @@ var FuzzerFuncs = fuzzer.MergeFuzzerFuncs(
 	schedulingfuzzer.Funcs,
 	discoveryfuzzer.Funcs,
 	flowcontrolfuzzer.Funcs,
+	apiextensionsfuzzer.Funcs,
```

Comment:
Looks like this line survived the rebase, it'll have to go to resolve the import issue.